Nodejs/Product/TargetsVsix/Microsoft.NodejsTools.targets

```diff
@@ -55,9 +53,18 @@
   </ItemGroup>
 
   <Target Name="CopyFilesToOutputDirectory"/>
+  
+  <Target Name="AfterClean">
+    <Exec Condition="'$(EnableTypeScript)' == 'true'" Command="&quot;$(Tsc)&quot; --build --clean" />
+  </Target>
+  
+  <Target Name="AfterCompile">
+    <Message Text="AfterCompile executed" />
+    <Exec Condition="'$(EnableTypeScript)' == 'true'" Command="&quot;$(Tsc)&quot; --build" />
+  </Target>
 
   <Target Name="CoreCompile">
-    <Error Condition="!Exists('$(VSToolsPath)\TypeScript\Microsoft.TypeScript.targets') And '$(EnableTypeScript)' == 'true' And '$(VisualStudioVersion)' != '16.0'" Text="TypeScript is not installed but TypeScript support is enabled in this project.  Please download and install TypeScript http://go.microsoft.com/fwlink/?LinkID=393159&amp;clcid=0x409."></Error>
+    <Error Condition="'$(EnableTypeScript)' == 'true' And !Exists('$(Tsc)')" Text="TypeScript compiler is not installed. Please execute 'npm install --save-dev typescript'"></Error>
```

Comment:
Should we tell the user to use the NPM UI on Visual Studio instead?

Reply:
Either way sounds fine to me.

Is there a way of localizing these strings? Are they English only?

Reply:
A quick search didn't throw any easy way of localizing. I can merge for now as is and follow up on a different issue for localizing the targets file if you're ok with it.